atomicapp/nulecule/main.py

```diff
@@ -90,6 +90,16 @@ def __init__(self, app_spec, destination=None, answers_file=None):
             if os.path.isfile(os.path.join(self.app_path, ANSWERS_FILE)):
                 self.answers_file = os.path.join(self.app_path, ANSWERS_FILE)
 
+        # TODO: put this in a better place in the future.
```

Comment:
Right.. I feel like if we add support to kubernetes for this same thing then we'll come up with the right place to "detect" this stuff and build out a framework for it.

Actually I'd like a single place where answers.conf + CLI args + defaults + detected environment overrides are all merged into one. Maybe I'll open an issue for that.